Sources/SteamPress/Controllers/BlogController.swift

```diff
@@ -30,12 +30,18 @@ struct BlogController {
         drop.group(pathCreator.blogPath ?? "") { index in
             index.get(handler: indexHandler)
             index.get(blogPostsPath, String.self, handler: blogPostHandler)
-            index.get(tagsPath, String.self, handler: tagViewHandler)
-            index.get(authorsPath, String.self, handler: authorViewHandler)
             index.get(apiPath, tagsPath, handler: tagApiHandler)
             index.get(blogPostsPath, handler: blogPostIndexRedirectHandler)
-            index.get(tagsPath, handler: allTagsViewHandler)
-            index.get(authorsPath, handler: allAuthorsViewHandler)
+
+            if (config.enableAuthorsPages) {
```

Comment:
Could you pass these `Bool`s into the initialiser for `BlogController` and do the Config checking in the `Provider`?